### ISSUE TITLE
User input and output to the prompt should be sanitized 

### ISSUE THREAD

**Issue by sberyozkin** [quarkiverse/quarkus-langchain4j#80]
By default, basic escaping of `<`, `>` and a few other characters should be done, but users should also be able to register per/post handlers for sanitizing if they would like to

**Comment by geoand** [comment]
Do you have examples of where this would be needed?

Also, are you interested in looking into this?

Thanks

**Comment by sberyozkin** [comment]
Yes, I'm creating issues based on https://developer.nvidia.com/blog/best-practices-for-securing-llm-enabled-applications/, which I'm finding as very well prepared, look forward to contributing

**Comment by sberyozkin** [comment]
Thinking more about it, users can just use Resteasy Reactive filters when necessary, but, some basic escaping may be worth doing by default on in/out, so keeping this issue open for now

**Comment by sberyozkin** [comment]
I wonder, if parameterizing the plugin input can be more useful, as shown in some demos, which covers the input, sanitizing the LLM output with the API key available seems questionable, just thinking aloud for now.
